test/chunking-form/samples/emit-file/deduplicate-assets/_config.js

```diff
@@ -1,16 +1,35 @@
+function asyncify(callback) {
```

Comment:
Instead of making this test random, which could potentially mean that the test is sometimes red and sometimes green but in an indeterministic way, why not work with e.g. fixed numbers as names but in a random order?
e.g. emit 1.txt, 2.txt (same source as 1.txt), 4.txt, 3.txt (same source as 4.txt) and assert that you get 1.txt and 3.txt as output (or 2 and 4, but I hope you see what I mean). Would also make the test nearly 100ms faster 😉

Reply:
I was trying to mock the use case mentioned in #4909. But I am fine with `emitFile` in a random order like you said.